web-components/src/components/user/UserInfo.tsx

```diff
@@ -90,7 +90,7 @@ export default function UserInfo({
       direction={direction}
       wrap="nowrap"
       sx={sx}
-      alignItems="center"
+      alignItems="top"
```

Comment:
This also has some impact on the user info on data stream/data post page
![image](https://github.com/user-attachments/assets/0d525839-2ad1-46b7-b936-9e71bf7f9bb4)

vs on dev:
![image](https://github.com/user-attachments/assets/7fd55e72-392a-493b-9b55-02e0c8f630fb)

see design: https://www.figma.com/design/Bksz1JeDYxQVIXdI46EgPT/Project-Posts?node-id=4710-119866&node-type=instance&m=dev

If we want to keep this as is (cc/ @erikalogie), it would be safer to pass `alignItems` as a prop to be able to customize it instead

Reply:
Agree, thanks for this @blushi - I have updated the code so now `alignItems` is passed down as a prop. 

@erikalogie if you want the user info on the data stream/data post page aligned at the top like in the stakeholders let me know.